artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/util/NettyReadable.java

```diff
@@ -222,7 +222,7 @@ public ReadableBuffer get(WritableBuffer target) {
 
    @Override
    public String toString() {
-      return buffer.toString();
+      return buffer != null ? buffer.toString() : this.getClass().getSimpleName();
```

Comment:
Seems like fixing the constuctor to reject the null buffer would be the actual fix. Essentially every method here NPEs, including this one, because its just not considered valid for wrapped buffer to be null. Should just enforce and properly test that.